osm/src/main/groovy/org/orbisgis/orbisanalysis/osm/utils/NominatimUtils.groovy

```diff
@@ -0,0 +1,169 @@
+package org.orbisgis.orbisanalysis.osm.utils
```

Comment:
Add header and authors

Reply:
header added, authors already present (see line 17-19)